### DISCUSSION TITLE
K Star/Cyg - fixed.

### DISCUSSION THREAD

**Discussion by DannyAblott** [StephanJoubert/home_assistant_solarman#362] [General]
I have a CYG inverter/battery system. I tried to implemt this in HA using the KStar. It didn't work at first (no entities). I noticed that the inverter definition file for the kstar has the wrong file extension. I changed it from "yml" to "yaml" and it now works fine.

**Comment by frsantos** [comment]
Thanks @DannyAblott, good catch. 

As it hasn't been release yet (it was added after version 1.4.3), I were just copying the definition content into the `custom_parameters.yml` file.

**Comment by Jay66Bee** [comment]
I'm rolling back to Solarman 1.4.3 with my KStar. There is a problem with 1.5.1 where the Solarman integration disconnects and fails to reconnect. This occurs once a week for me. More frequently for others. The only solution (for now) is to restart HA, which is a nuisance.

I don't think it's KStar specific as a variety of inverter users are having the same problem with 1.5.1

I shall do without the "benefits" of 1.5.1 until it is stable.
